package.json

```diff
@@ -21,7 +21,7 @@
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint test:*",
+    "test": "npm-run-all test:*",
     "test:ember": "ember test",
     "test:babel": "vitest --run",
     "test:ember-compatibility": "ember try:each"
```

Comment:
we don't want ember-compatibility to run with `test`

I think for ci.yml, we should have it run as a separate command, rather than try to make npm-run-all do it all. We want compatibility to run in try-scenarios, and not with _every_ test job